demos/CMakeLists.txt

```diff
@@ -52,5 +52,21 @@ set_target_properties(Demo_CornellBox PROPERTIES FOLDER "Demos")
 # Properties->Linker->Input->Additional Dependencies
 target_link_libraries(Demo_CornellBox ${OpenCV_LIBS} ${CORE_LIB})
 
+# ================================================= DEMO SPOT LIGHT ============================================
+add_executable(Demo_SpotLight "Demo SpotLight.cpp")
+add_dependencies(Demo_SpotLight core)
+
+source_group("Source Files" FILES "Demo SpotLight.cpp")
+
+set_target_properties(Demo_SpotLight PROPERTIES PROJECT_LABEL "Demo SpotLight")						# in Visual Studio
+set_target_properties(Demo_SpotLight PROPERTIES OUTPUT_NAME "Demo SpotLight")
+set_target_properties(Demo_SpotLight PROPERTIES FOLDER "Demos")
+
+# Properties->Linker->Input->Additional Dependencies
+target_link_libraries(Demo_SpotLight ${OpenCV_LIBS} ${CORE_LIB})
+# --------
+#install
+install(TARGETS Demo_SpotLight RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+
 #install
 install(TARGETS Demo_CornellBox RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)

```

Comment:
It looks like this line belongs to the previous block (DEMO CORNELL BOX)

Reply:
Fixed